heal-remedy.lic

```diff
@@ -52,38 +52,38 @@ def wounds
   hurt = hurt.map { |raw| wound_grouping[raw] || raw }.uniq
   scar = hurt.map { |raw| scar_grouping[raw] || raw }.uniq
   if $debug_mode_hr
-    echo 'Wounds: '
+    DRC.message("Wounds: ")
```

Comment:
Aren't a lot of these `message`s redundant? I.e. it echoes `hurt` on line 57 and then again on line 68. Same with `scar` on lines 64 and 76-77.

Reply:
The key changes for wounds and then for scars. May seem redundant but it was a sanity check to make sure everything was resolving properly through the different steps.